__tests__/app.test.js

```diff
@@ -38,6 +38,75 @@ describe('/api/topics', () => {
 		});
 	});
 });
+describe('/api/articles', () => {
```

Comment:
There's a couple of tests I would add here:
- what happens if we have a valid topic but it has no articles (this would still give us a 200)
- 404 test for a topic that was not found